Source/WebCore/testing/ServiceWorkerInternals.cpp

```diff
@@ -183,9 +183,9 @@ void ServiceWorkerInternals::lastNavigationWasAppInitiated(Ref<DeferredPromise>&
 RefPtr<PushSubscription> ServiceWorkerInternals::createPushSubscription(const String& endpoint, std::optional<EpochTimeStamp> expirationTime, const ArrayBuffer& serverVAPIDPublicKey, const ArrayBuffer& clientECDHPublicKey, const ArrayBuffer& auth)
 {
     auto myEndpoint = endpoint;
-    Vector<uint8_t> myServerVAPIDPublicKey { static_cast<const uint8_t*>(serverVAPIDPublicKey.data()), serverVAPIDPublicKey.byteLength() };
-    Vector<uint8_t> myClientECDHPublicKey { static_cast<const uint8_t*>(clientECDHPublicKey.data()), clientECDHPublicKey.byteLength() };
-    Vector<uint8_t> myAuth { static_cast<const uint8_t*>(auth.data()), auth.byteLength() };
+    auto myServerVAPIDPublicKey = serverVAPIDPublicKey.toVector();
+    auto myClientECDHPublicKey = clientECDHPublicKey.toVector();
+    auto myAuth = auth.toVector();
```

Comment:
Now that these are so much less wordy, I think it would be better to just call them below and not have local variables or `WTFMove` involved.